src/main/scala/com/kinja/config/ConfigError.scala

```diff
@@ -0,0 +1,25 @@
+package com.kinja.config
+
+/**
+ * The types of errors that a BootupErrors can catch
+ */
+sealed trait ConfigError {
```

Comment:
It helps type inference if this extends `Serializable` and `Product` since the case classes extending it do (this makes `ConfigError` into the least-upper-bound instead of `ConfigError with Serializable with Product`.